chpl/chpl-service/src/main/java/gov/healthit/chpl/domain/CertificationResult.java

```diff
@@ -213,6 +219,14 @@
     @Singular
     private List<MacraMeasure> g2MacraMeasures = new ArrayList<MacraMeasure>();
 
+    /**
+     * NEED TO GET THIS TEXT
```

Comment:
Still need to

Reply:
Does this text get inserted anywhere? If it's displayed to an end user we need it to be filled in before it goes to PROD

Reply:
We should get this text. It gets put in the XSD as a comment that describes the field.